src/pages/Profile.tsx

```diff
@@ -1,6 +1,112 @@
 import React from 'react';
-import { Text } from 'react-native';
+import { useContext } from 'react';
+import { AuthContext } from '../context/AuthContext';
+import FeatherIcon from 'react-native-vector-icons/Feather';
+import { TouchableOpacity, StyleSheet, Text, View } from 'react-native';
+import { NavigationProp } from '@react-navigation/core';
+import { CustomButton } from '../components/CustomButton';
 
-export const Profile = () => {
-  return <Text>Profile</Text>;
+interface ProfileProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  navigation: NavigationProp<any, any>;
+}
+
+export const Profile = ({ navigation }: ProfileProps) => {
+  const redirectToResetPassword = () => {
+    navigation.navigate('ResetPassword');
+  };
+  const { user } = useContext(AuthContext);
+  return (
+    <View style={Styles.container}>
+      <View style={Styles.header}>
+        <Text style={Styles.headerTitle}>PROFILE</Text>
+      </View>
+      <View style={Styles.main}>
+        <Text style={[Styles.mainTitle, Styles.mainTitleBold]}>
+          {user?.username}
+        </Text>
+        <Text style={Styles.mainTitle}>{user?.email}</Text>
+        <TouchableOpacity
+          style={Styles.linkCard}
+          onPress={redirectToResetPassword}
+        >
+          <View style={Styles.groupIconText}>
+            <FeatherIcon name={'lock'} size={20} color={'black'} />
```

Comment:
1. The icons and the texts are using different text colors. I think it's better to use the same for all, try with "#5c5c5c".

2. Can you separate this as a reusable component? Maybe the component can receive the icon name, the text to display and the name of the view to redirect.